src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1,4 +1,45 @@
-
-// Generated from SparqlAutomatic.g4 by ANTLR 4.9.2
+// Copyright 2021, University of Freiburg,
+// Chair of Algorithms and Data Structures
+// Author: Hannah Bast <bast@cs.uni-freiburg.de>
 
 #include "SparqlQleverVisitor.h"
+
+#include <string>
+#include <vector>
+
+// ___________________________________________________________________________
+antlrcpp::Any SparqlQleverVisitor::processIriFunctionCall(
+    std::string iri, std::vector<SparqlQleverVisitor::ExpressionPtr> argList) {
```

Comment:
You only look at the `iri`, so it should be a `const string&`.

Reply:
Done!